WalletWasabi/Blockchain/Transactions/TransactionFactory.cs

```diff
@@ -232,41 +231,21 @@ public TransactionFactory(Network network, KeyManager keyManager, ICoinsView coi
 				builder = builder.AddKeys(signingKeys.ToArray());
 				builder.SignPSBT(psbt);
 
-				var isPayjoin = false;
-
 				// Try to pay using payjoin
 				if (payjoinClient is not null)
 				{
 					psbt = TryNegotiatePayjoin(payjoinClient, builder, psbt, changeHdPubKey);
-					isPayjoin = true;
 					psbt.AddKeyPaths(KeyManager);
 					psbt.AddPrevTxs(TransactionStore);
 				}
 
 				psbt.Finalize();
 				tx = psbt.ExtractTransaction();
 
-				var checkResults = builder.Check(tx).ToList();
```

Comment:
Why did you also remove this and later on its pair:

```cs
if (checkResults.Count > 0)
{
		throw new InvalidTxException(tx, checkResults);
}
```

Reply:
It was only used to add the `Fees different` error and then to throw the exception.

Reply:
Nah, think again 😄

```cs
var checkResults = builder.Check(tx).ToList();
if (checkResults.Count > 0)
{
	throw new InvalidTxException(tx, checkResults);
}
```

Reply:
Oh, I didn't pay attention to `builder.Check(tx).ToList()`.

Reply:
Fixed. https://github.com/zkSNACKs/WalletWasabi/pull/6909/commits/4b8381205670ab5c6847b7c2e1d5759ca5c84fa2